packages/api/src/controllers/asset.ts

```diff
@@ -135,6 +138,25 @@ async function validateAssetPayload(
   };
 }
 
+function validateAssetPlaybackPolicy(playbackPolicy: Asset["playbackPolicy"]) {
+  if (playbackPolicy?.type == "lit_signing_condition") {
+    if (!playbackPolicy.unifiedAccessControlConditions) {
+      throw new UnprocessableEntityError(
+        `playbackPolicy.unifiedAccessControlConditions is required when using lit_signing_condition`
+      );
+    }
+    if (!playbackPolicy?.resourceId) {
+      // TODO: Generate resourceId if not present
+    }
+  }
+  if (playbackPolicy?.type == "jwt") {
```

Comment:
Perhaps check that it's not one of the supported ones instead
```suggestion
  if (!["public", "lit_signing_conditions"].includes(playbackPolicy?.type)) {
```